docs/install.md

```diff
@@ -0,0 +1,30 @@
+# MlAgility Installation Guide
```

Comment:
Link on //readme.md and the benchit user guide

Reply:
Added!